File: cc/009.cc
#include <iostream>
using namespace std;

class Solution
{
  public:
    int jumpFloorII(int number)
    {
        if (number == 0)
            return 0;
        if (number == 1)
            return 1;
        return 2 * this->jumpFloorII(number - 1);
    }
};
int main(int argc, char const *argv[])
{
    Solution *s = new Solution();
    cout << s->jumpFloorII(9) << endl;
    cin.get();
    return 0;
}

File: python/009.py
# -*- coding: utf-8 -*-
# /usr/bin/python3
# Creator: Hemajun
# Description: 009


class Solution:
    def jumpFloorII(self, number):
        if number == 0 or number == 1:
            return number
        return 2 * self.jumpFloorII(number-1)
